src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/PropertySymbolReferenceFinder.cs

```diff
@@ -155,11 +155,11 @@ private static bool IsForEachProperty(IPropertySymbol symbol)
             return nameReferences.Concat(forEachReferences, indexerReferences, suppressionReferences);
         }
 
-        private static Task<ImmutableArray<Document>> FindDocumentWithElementAccessExpressionsAsync(
```

Comment:
What's the difference between "WithElementAccess" and "WithExplicitOrImplicitElementAccess"?

Reply:
After reading more I'm guessing "WithElementAccess" is only for explicit cases?

Reply:
Correct.